app/_gateway_entities/license.md

```diff
@@ -22,8 +18,217 @@ schema:
+* The Admin API allows OSS features to continue working and configured {{site.ee_product_name}} features to continue operating in read-only mode.
+* Proxy traffic, including traffic using Enterprise plugins, continues to be processed as if the License hadn't expired.
+* Other Enterprise features aren't accessible.
+* There may be some Enterprise features that are still writable, but they may also change later, so don't rely on this behavior.
+
+The behavior of the different deployment modes is as follows:
+
+- **Traditional:** Nodes will be able to restart/scale as needed.
+- **Hybrid:** Existing data planes or new data planes **can accept** config from a control plane with an expired License.
+- **DB-less and KIC:** New nodes **cannot** come up, restarts will break.
+
+You can update your License with a `PUT` request to the [`/license/{license-id}` Admin API endpoint](/api/gateway/admin-ee/).
+
+### Logs
+
+{{site.base_gateway}} logs the License expiration date on the following schedule:
+* 90 days before: `WARN` log entry once a day
+* 30 days before: `ERR` log entry once a day
+* At and after expiration: `CRIT` log entry once a day
+
+## Monitor License usage and deployment information
+
+A license report contains information about your {{site.base_gateway}} database-backed deployment, including License usage and deployment information. The license report module manually generates a report containing usage and deployment data by sending a request to the `/license/report` endpoint. Share this information with Kong Support to perform a health-check analysis of product utilization and overall deployment performance to ensure your organization is optimized with the best License and deployment plan for your needs.
+
+The license report **does not**:
+*   Automatically generate a report or send any data to any Kong servers
+*   Track or generate any data other than the data that is returned in the response after you send a request to the endpoint
+
+{:.important}
+> **Important:** The license report functionality cannot be used in a DB-less deployment.
+
+### Get a license report
+
+| Method | Command |
+|--------|----------|
+| JSON | `curl -i -X GET http://localhost:8001/license/report` |
+| TAR | `curl http://localhost:8001/license/report -o response.json && tar -cf report-$(date +"%Y_%m_%d_%I_%M_%p").tar response.json` |
+
+Here's an example license report:
+
+```json
+{
+   "counters": [
+       {
+           "bucket": "2021-12",
+           "request_count": 0
+       }
+   ],
+   "db_version": "postgres 9.6.19",
+   "kong_version": "3.9.0.0",
+   "license_key": "ASDASDASDASDASDASDASDASDASD_ASDASDA",
+   "rbac_users": 0,
+   "services_count": 0,
+   "system_info": {
+       "cores": 4,
+       "hostname": "13b867agsa008",
+       "uname": "Linux x86_64"
+   },
+   "workspaces_count": 1
+}
+```
+
+### License report reference
```

Comment:
Write this table in another file and then reference it here.

Reply:
![image](https://github.com/user-attachments/assets/5d959d53-810f-4836-b169-6104ac37cbe2)
The bullets render poorly, The stuff in the table seems to be the fields in the schema for simplicity I would just link to the schema
https://docs.konghq.com/gateway/api/admin-ee/latest/#/licenses/get-license-report

Reply:
Appreciate that you called this out. I found the schema was missing some of these values, so I'll make a GH issue to fix the schema.